GameData/RealismOverhaul/RO_SuggestedMods/Bluedog_DB/Able/RO_Able.cfg

```diff
@@ -0,0 +1,179 @@
+//	================================================================================
+//	Able Configs
+//	================================================================================
+
+
+// Thor Able Avionics
+@PART[bluedog_ThorAble_Guidance]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.6
+	@title = Thor Able Guidance Unit
+	@manufacturer = Aerojet
+	@description = Guidance core for the Thor Able upper stage as used on Thor Able and early Delta rockets. Goes on top of the Able second stage tank.
+	@mass = 0.11
+	@MODULE[ModuleCommand]
+	{
+		@RESOURCE[ElectricCharge]
+		{
+			@rate = 0.05
+		}
+	}
+	@MODULE[ModuleSAS]
+	{
+		%SASServiceLevel = 1
+	}
+	!MODULE[ModuleFuelTanks] {}
+	MODULE
+	{
+		name = ModuleFuelTanks
+		volume = 1
+		basemass = -1
+		type = ServiceModule
+		TANK
+		{
+			name = ElectricCharge
+			amount = 2000
+			maxAmount = 2000
+		}
+	}
+}
+
+
+// Thor Able
+@PART[bluedog_ThorAble_Tank]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.6
+	@title = Able Tank (Delta A)
+	@manufacturer = Aerojet
+	@description = Lightweight 1m fuel tank for the Thor-Able upper stage.
+	@mass = 0.25
+	maxTemp = 773.15
+	skinMaxTemp = 873.15
+	!MODULE[ModuleB9PartSwitch]:HAS[#moduleID[volumeSwitch]] {}
+	!RESOURCE,* {}
+	MODULE
+	{
+		name = ModuleFuelTanks
+		type = ServiceModule
+		volume = 1320
+		basemass = -1
+	}
+	RESOURCE

```

Comment:
There's no reason to make this a stock tank? Just put it in an MFT tank.

Also, Able used Helium-based RCS